payload/src/dynamics.ts

```diff
@@ -147,6 +150,27 @@ export function FormSubmit(self: HTMLElement, e: Event) {
     return false
 }
 
+function TriggerLoaders(self: HTMLElement) {
```

Comment:
Function must to be underscored: `_TriggerLoaders`  
This function is for our internal use, not for public